client/info.go

```diff
@@ -0,0 +1,29 @@
+package client
+
+import (
+	"github.com/mk6i/retro-aim-server/wire"
+	"github.com/mk6i/smarter-smarter-child/config"
+)
+
+// Set the bot's profile
+func SetInfo(flapc FlapClient, config config.Config) error {
```

Comment:
To keep things consistent, let's keep the function un-exported (`setInfo`) and move it into `chat.go`

Reply:
And something like `sendInfoSNAC` might be a bit more descriptive.

Reply:
I didn't include it in chat.go because it's unrelated to conversation handling. I'll move it and revisit if chat.go gets too unwieldy in the future.